src/components/AxisCollection/YAxis.js

```diff
@@ -193,7 +210,7 @@ class YAxis extends Component {
             <g key={+v} opacity={1} transform={`translate(0, ${scale(v)})`}>
               <line {...lineProps} />
               <text className="tick-value" {...textProps}>
-                {tickFormatter(v)}
+                {tickFormatter(v).toFixed(maxDecimals)}
```

Comment:
We shouldn't be calling `toFixed` on the result of tickFormatter. For example, what if I return a string?

For example: https://griff-react-pr-230.surge.sh/?selectedKind=Scatterplot&selectedStory=Custom%20tick%20formatting&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
Changed to let tickFormatter take the array as a param.